docs/migration guide/migration_guide.md

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
1. The folder name should be migration_guide not "migration guide"
2. The titles in the file should be:
> v0.0.Y
>> Do W
>> Do Z
>> Do K
> v0.0.X
...

The latest version should be on top.

Reply:
@paganotoni I just applied your suggestions